frontend/src/tests/components/eMIB/ResponseItem.test.js

```diff
@@ -0,0 +1,207 @@
+  const forward = <span className="fas fa-share-square" />;
+
+  //reply
+  expect(replyWrapper.containsMatchingElement(reply)).toEqual(true);
+  expect(replyWrapper.containsMatchingElement(replyAll)).toEqual(false);
+  expect(replyWrapper.containsMatchingElement(forward)).toEqual(false);
+
+  //reply all
+  expect(replyAllWrapper.containsMatchingElement(reply)).toEqual(false);
+  expect(replyAllWrapper.containsMatchingElement(replyAll)).toEqual(true);
+  expect(replyAllWrapper.containsMatchingElement(forward)).toEqual(false);
+
+  //forward
+  expect(forwardWrapper.containsMatchingElement(reply)).toEqual(false);
+  expect(forwardWrapper.containsMatchingElement(replyAll)).toEqual(false);
+  expect(forwardWrapper.containsMatchingElement(forward)).toEqual(true);
+});
+
+it("renders email's header (with and without cc, since it is optional)", () => {
+  //with cc field
+  const withCcWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  //without cc field
+  const withoutCcWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  const headerContentWithCc = (
+    <div>
+      <div>
+        <p>
+          {LOCALIZE.emibTest.inboxPage.emailResponse.description}
+          <>
+            <span className="fas fa-reply" />
+            <span>{LOCALIZE.emibTest.inboxPage.emailResponse.responseType.reply}</span>
+          </>
+        </p>
+        <p>
+          <span className="font-weight-bold">
+            {LOCALIZE.emibTest.inboxPage.emailResponse.to}&nbsp;
+          </span>
+          <span>to</span>
+        </p>
+        <p>
+          <span className="font-weight-bold">
+            {LOCALIZE.emibTest.inboxPage.emailResponse.cc}&nbsp;
+          </span>
+          <span>cc</span>
+        </p>
+      </div>
+    </div>
+  );
+
+  const headerContentWithoutCc = (
+    <div>
+      <div>
+        <p>
+          {LOCALIZE.emibTest.inboxPage.emailResponse.description}
+          <>
+            <span className="fas fa-reply" />
+            <span>{LOCALIZE.emibTest.inboxPage.emailResponse.responseType.reply}</span>
+          </>
+        </p>
+        <p>
+          <span className="font-weight-bold">
+            {LOCALIZE.emibTest.inboxPage.emailResponse.to}&nbsp;
+          </span>
+          <span>to</span>
+        </p>
+        <p>
+          <span className="font-weight-bold">
+            {LOCALIZE.emibTest.inboxPage.emailResponse.cc}&nbsp;
+          </span>
+          <span />
+        </p>
+      </div>
+    </div>
+  );
+
+  expect(withCcWrapper.containsMatchingElement(headerContentWithCc)).toEqual(true);
+  expect(withoutCcWrapper.containsMatchingElement(headerContentWithoutCc)).toEqual(true);
+});
+
+it("renders email's response", () => {
```

Comment:
This one isn't necessary - there's no functional pieces to test here.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/172/commits/65fa676ee1553d366b684d85bae2a6d108d7851a